app/controllers/line_items_controller.rb

```diff
@@ -0,0 +1,58 @@
+class LineItemsController < ApplicationController
+  include CurrentCart
+  before_action :set_cart, only: [:create, :destroy]
+  before_action :set_line_item, only: [:update, :destroy]
```

Comment:
@b-artem it should be processed via cancancan using `load_and_authorize_resource`